libs/user_extensions/include/TTAlpsHistogramFiller.hpp

```diff
@@ -13,9 +13,6 @@ class TTAlpsHistogramFiller {
   TTAlpsHistogramFiller(std::shared_ptr<HistogramsHandler> histogramsHandler_);
   ~TTAlpsHistogramFiller();
 
-  void FillLeadingPt(const std::shared_ptr<Event> event, std::string histName, const HistogramParams &params);
-  void FillAllSubLeadingPt(const std::shared_ptr<Event> event, std::string histName, const HistogramParams &params);
```

Comment:
Oh, btw. I noticed that leading/subleading histograms are not filled anymore, and now you remove these methods - do we not want to have these plots? Or are they now filled somewhere else?